src/components/dialog.js

```diff
@@ -0,0 +1,33 @@
+import * as RadixDialog from "@radix-ui/react-dialog";
+import { forwardRef } from "react";
+
+// FIXME
```

Comment:
Would it make sense to add some context on what the issue is/what should be fixed?

Reply:
Yea, good call. I realize this is not very descriptive for anyone 😬